app/config/config.yml

```diff
@@ -222,51 +222,8 @@ sonata_doctrine_phpcr_admin:
     templates:
         form:
             - admin/form_admin_fields.html.twig
-    document_tree_defaults: [locale]
     document_tree:
-        Doctrine\ODM\PHPCR\Document\Generic:
-            valid_children:
-                - all
```

Comment:
did we get completely rid of the feature to define what can be where in the tree? or did it just move elsewhere?